decidim-core/app/commands/decidim/destroy_account.rb

```diff
@@ -37,6 +37,8 @@ def destroy_user_account!
       current_user.name = ""
       current_user.nickname = ""
       current_user.email = ""
+      current_user.personal_url = ""
+      current_user.about = ""
```

Comment:
I'm hesitant on this change. Like I agree that we should do so, but I have doubts if this actually need a bit more work. Like: 

1. adding a rake task for deleting these fields from already deleted participants
2. adding a note in the Releases Notes
3. changing the label to `type: fix` so we can backport it

What do you think?

Reply:
Yes, it makes sense :)

Reply:
Done in 100b200ce5